scripts/variantstore/variant_annotations_table/README.md

```diff
@@ -47,6 +47,18 @@ Once those inputs have been created, they can be used by the GvsCreateVATfromVDS
 | output_path          | the path that the temp files will be stored           |
 | project_id           | the name of the google project containing the dataset |
 
+### Using the Nirvana reference image in Terra (AoU Echo and later callsets)
+
+The Variants team created a Cromwell reference image containing all reference files used by Nirvana 3.18.1. This is
+useful to avoid having to download tens of GiBs of Nirvana references in each shard of the scattered `AnnotateVCF`.
+In order to use this reference disk, the 'Use reference disk' option in Terra must be selected as shown below:
+
+![Terra Use reference disks](Reference%20Disk%20Terra%20Opt%20In.png)
+
+Note: this Nirvana reference image was not available in time for it to be used to create the Delta VAT. The
+`GvsCreateVATfromVDS` WDL has been speculatively updated to take advantage of references loaded from an attached
```

Comment:
👍

Reply:
I think this could be a teeny bit more aggressive ("hey, make sure you run this on a test set before Echo, dummy!") haha

Reply:
Yeah this dummy will be testing his changes with Quickstart data as soon as the [WX-910 fixes](https://github.com/broadinstitute/cromwell/pull/6989) hit production, hopefully early next week with the monolith release.